IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+
+This tutorial will give you a walkthrough on how certain functions of HTML work and why they are important, all while learning how to build a cool mini-city!
+
+## Setup
+Open any HTML/CSS editor. A good browser-based editor is:
+https://www.w3schools.com/tryit/tryit.asp?filename=tryhtml_default.
+Follow along with this tutorial, and you will be one step closer to building your first mini-city!
+
+## Walkthrough
+
+To build our mini-city, we will need to use tools such as HTML, CSS, and Javascript. This tutorial will go over the HTML portion of the mini-city. You can think of HTML as the foundation of the entire project! If we work to build a strong foundation, our mini-city will look fantastic!
+
+Our first step when using HTML is to establish a “!DOCTYPE html” function. This simply means that all of our code below this will be rendered as an HTML document.
+So, your first line should look like this: <img src=https://lh3.googleusercontent.com/3-aYKYNPyv2Ie-XOajV4_q8EM1PDzeSIxRDKeKtAsSxj_Cy9gSzlhYjDZ_d2UGupdsr5DvKaZy0tfP6NpqwOm8QXHNIIai5XspYBKACo_MODMXnfLni4-VK4KaNcdq2KMLWEGN-A>
+
+
+Got it? Great!
+
+Next, we have to actually create our HTML tag. This just means that we are beginning to write our HTML code, but first we must actually tell our computer that we will be writing in HTML. To do this, simply type “html” on your second line. Your code should look like this so far: <img src=https://lh4.googleusercontent.com/xcemOKcTNUUlo5yAwQvAyAaMhLiwskacjc1LTDy0P0yoaSUpssBCjlURYTfSYsw-yHPBpXGgJCK_Y76lxqLPbjmiEx7WSMxJ5v-OclRHJo1J7YkQNaqI9sR-el_PM8qnaUp0rjvt>
+
+Now that we have established the fact that we will be writing in HTML, it’s time to get into the fun stuff. We start by writing “head” in our third line of code. This tells our editor that the information for our header will be stored here.
+
+After we write our “head” tag, we know that we should give our project a title. Let’s title our project “My Mini City”. This is how your code should look now: <img src=https://lh3.googleusercontent.com/1ZecOiCyLV4wjTHfQsGzQSuJJGePIZp46IQkE29VVEmbHz1WUeS5tsDMs0iiKPVIFjCyic2dk49RRXEbnbwYrJFjGgykmT4rVDqWedqytukmAUpdbSiDKXYdvogTciuwNopK4Vzl>
+
+You may notice that there is a /title tag after our title’s name. This is called a closing tag. Closing tags have a “/” before they state which tag they are closing. This just tells our editor that we are no longer editing this tag.
+
+To create our body section, simply type body on your next line.
+
+Now, let’s make our classes! Let’s think about what we want our city to look like.
+We know that we want the following objects in our city:
+The ground
+A skyscraper
+A smaller building
+A road
+A bridge
+An ocean
+A waterfall
+A car
+
+In order to eventually create these objects, we have to make classes for them. Classes are basically like blank notebooks that we use to establish the fact that we are going to create an object. Then, we actually create it in CSS!
```

Comment:
Is it possible for you to expand on your analogy of "blank notebooks"? I don't quite know what that means.